app/models/vehicle.rb

```diff
@@ -0,0 +1,10 @@
+class Vehicle < ActiveRecord::Base
+  STATUS = {
+    :stock => I18n.t('stock', scope: 'enumerize.vehicle.status'),
+    :sold => I18n.t('sold', scope: 'enumerize.vehicle.status')
+  }
+
+  def status
+    I18n.t(read_attribute(:status), scope: 'enumerize.vehicle.status').capitalize
+  end
```

Comment:
Commonly I use `enumerize`, but how I use this just here I not change yet.
I'm walking like baby steps. :baby: